src/compas/datastructures/mesh/duality.py

```diff
@@ -21,6 +22,9 @@ def mesh_dual(mesh, cls=None):
     cls : Type[:class:`~compas.datastructures.Mesh`], optional
         The type of the dual mesh.
         Defaults to the type of the provided mesh object.
+    boundary: float, optional
+        boundary mode for the dual mesh
+        Default mode is 0, not create faces on boundaries
```

Comment:
Boundary mode for the dual mesh
Default mode is 0, which will not add new faces to the boundaries